main.go

```diff
@@ -83,5 +83,6 @@ func LoadData() [][]string {
 	cr := csv.NewReader(r)
 	records, _ := cr.ReadAll()
```

Comment:
Guessing that is an error raised.  You should make sure you catch-and-handle all errors to prevent weird issues.  Granted, I don't know the full context of what this webserver is doing.